bin/webpack.js

```diff
@@ -1,19 +1,59 @@
 #!/usr/bin/env node
 
+const {
+	exec
+} = require("child_process");
+const inquirer = require("inquirer");
+
+function runCommand(command) {
+	exec(command, (error, stdout, stderr) => {
+		if(!error) {
+			console.log("webpack-cli installed successfully");
+			return true;
+		}
+		console.log("failed to install webpack-cli");
+		console.error(stderr);
+		return false;
+	});
+}
+
 let webpackCliInstalled = false;
 try {
 	require.resolve("webpack-cli");
 	webpackCliInstalled = true;
-} catch(e) {
+} catch(err) {
 	webpackCliInstalled = false;
 }
 
-if(webpackCliInstalled) {
-	require("webpack-cli"); // eslint-disable-line node/no-missing-require, node/no-extraneous-require, node/no-unpublished-require
+if(!webpackCliInstalled) {
+	const path = require("path");
+	const fs = require("fs");
+	const isYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
+
+	let command;
```

Comment:
can we do this after the question, inside the if condition ?

Reply:
Yeah, of course, that would scope these things much more, which makes much more sense. I will apply those refactors ASAP.